python/sidecar/http.py

```diff
@@ -31,54 +20,54 @@ async def sidecar_index():
 
 
 @sidecar_api.post("/ingest")
-async def upload() -> IngestResponse:
-    # response = sidecar.ingest.run_ingest()
+async def http_ingest() -> IngestResponse:
+    # response = ingest.run_ingest()
     raise NotImplementedError("Ingest is not yet implemented")
 
 
 @sidecar_api.get("/ingest_status")
-async def get_statuses() -> IngestStatusResponse:
-    response = sidecar.ingest.get_statuses()
+async def http_get_statuses() -> IngestStatusResponse:
```

Comment:
Why prefix these with `http`?

Reply:
I've had to prefix the `chat` and then I’ve used the same rule for all the other methods. A bit of OCD here I must admit :-).

Also, this is something I do in my projects when implementing HTTP apis, where I normally use `*Handler` (ex: `chatHandler`). I can use that convention if you want.

Reply:
I would prefer to not prefix them all with http_ since they are already namespaced by nature of being in the http module. If there is a name collision for chat I would just do either:
```
from sidecar import chat as chat_

or 

from sidecar import chat as chat_module
```

Reply:
Ok. I will make that change instead.